src/cms/content-staging-scheduled-update.md

```diff
@@ -3,107 +3,61 @@ ee_only: true
 title: Scheduling an Update
 ---
 
-The following example shows how to schedule a temporary price change for a product. You will learn how to schedule and preview changes, and also how to view scheduled updates on the calendar. Although this example includes only a single change, a campaign might include multiple changes to products, price rules, CMS pages, and other entities that are scheduled to take place at the same time.
+The following example shows how to schedule a temporary price change for a product. This includes scheduling and previewing changes, as well as viewing scheduled updates on the calendar. Although this example includes only a single change, a campaign might include multiple changes to products, price rules, CMS pages, and other entities that are scheduled to take place at the same time.
 
-## To schedule an update to a product:
+## Schedule an update to a product
 
-1. From the **Catalog** grid, open a product in edit mode.
+1. From the _Products_ grid, open a product in edit mode.
 
-1. In the **Scheduled Changes** box at the top of the page, click <span class="btn">Schedule New Update</span>.
+1. In the _Scheduled Changes_ box at the top of the page, click <span class="btn">Schedule New Update</span>.
 
-    ![]({% link images/images-ee/content-staging-product-schedule-new-update.png %}){: .zoom}
+   ![]({% link images/images-ee/content-staging-product-schedule-new-update.png %}){: .zoom}
 
-1. With the **Save as a New Update** option selected, do the following:
+1. With the **Save as a New Update** option selected, Set the basic parameters for the update:
 
-    - In the **Update Name** field, enter a name for the new content staging campaign.
+   - For **Update Name**, enter a name for the new content staging campaign.
 
-    - Enter a brief **Description** of the update and how it is to be used.
+   - Enter a brief **Description** of the update and how it is to be used.
 
-    - Use the Calendar (![]({% link images/images/btn-calendar.png %}){: .Inline}) to choose the **Start Date** and **End Date** for the campaign.
+   - Use the Calendar (![]({% link images/images/btn-calendar.png %})) tool to choose the **Start Date** and **End Date** for the campaign.
 
-    To create an open-ended campaign, leave the End Date blank. For this example, the campaign is scheduled to begin the minute the sun reaches the northernmost point of the equator. This year, the summer solstice takes place on June 20th, at 3:34 AM PST.
+      To create an open-ended campaign, leave the end date blank. For this example, the campaign is scheduled to begin at the stroke of midnight for the new year, January 1, 2021 at 12:00 AM PST.
```

Comment:
recommend treating End Date as a field name.  capped and bold?

Reply:
I chose to just treat it as not the proper name here. It would mean too much bold/italic and the emphasis is not on the specific fields but in how they define their date/time range.